### PR TITLE
Merge exceptions

### PR DESCRIPTION
**Pull request by Wraith2**
This combines 3 exceptions from both codebases into a single definition. It feels like generally the simplification work I'm looking at has already been done in the netcore codebase, are the netfx and netcore codebases from different times?

### CODE REVIEW

File: src/Microsoft.Data.SqlClient.sln [hunk 1]
```diff
@@ -876,16 +876,16 @@ Global
 		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp2.1-Release|x86.Build.0 = netcoreapp2.1-Release|Any CPU
 		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|Any CPU.Build.0 = Debug|Any CPU
-		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|x64.ActiveCfg = netcoreapp3.1-Debug|x64
-		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|x64.Build.0 = netcoreapp3.1-Debug|x64
-		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|x86.ActiveCfg = netcoreapp3.1-Debug|x86
-		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|x86.Build.0 = netcoreapp3.1-Debug|x86
+		{9073ABEF-92E0-4702-BB23-2C99CEF9BDD7}.netcoreapp3.1-Debug|x64.ActiveCfg = netcoreapp2.1-Debug|x64
```
Comment (karinazhou) [inline src/Microsoft.Data.SqlClient.sln:879]: Please revert the changes in this file.

Comment (Wraith2) [reply src/Microsoft.Data.SqlClient.sln:879]: Is there a way to stop visual studio changing the solution? even with a clean repo anytime I open the project it changes the config which ends up with me needing to constantly remove it from pr's (or forgetting as I did this time).
